tests/graphql/client.spec.ts

```diff
@@ -552,27 +551,6 @@ describe("createSubscriptionClient", () => {
       });
       expect(subscriptionClient).to.equal(fakeGraphqlWsClient);
     });
-
-    describe("when executing in a browser environment", () => {
-      before(() => {
-        (global as any).window = {
-          WebSocket: {...ws, fake: true},
-        };
-      });
-
-      after(() => {
-        (global as any).window = undefined;
-      });
-
-      it("should use the native browser WebSocket implementation", () => {
-        createClientStub.resetHistory();
-
-        client.graphQL.createSubscriptionClient();
-
-        expect(createClientStub.callCount).to.equal(1);
-        expect(createClientStub.getCall(0).args[0].webSocketImpl.fake).to.equal(true);
-      });
-    });
```

Comment:
🤔 why we are removing test? we can still verify if `isomorphic-ws` using native WebSocket with this

Reply:
This test case was specifically for `typeof window === "undefined"` condition that was in our code base, but now it's removed and it's now internal logic of `isomorphic-ws`. They actually handle it in more graceful way by having `browser` entry point in `package.json` that loads very tiny JS wrapper to normalize native websocket and for node.js it loads completely different file.

Anyway, I've extendede previous test case to verify that we pass `WebSocket` from `isomorphic-ws` to `createClient`: d5600cc